framework/android/kodein-di-framework-android-x-viewmodel-savedstate/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id("org.kodein.library.android")
+}
+
+dependencies {
+    api(project(":framework:android:kodein-di-framework-android-x"))
+    implementation("androidx.core:core-ktx:1.6.0")
+    implementation("androidx.appcompat:appcompat:1.3.0")
+    implementation("androidx.activity:activity-ktx:1.2.3")
```

Comment:
this one seems out of scope.

Reply:
I will review and touch this up or report back if there is an issue.